addons/account_peppol/wizard/account_move_send.py

```diff
@@ -111,6 +111,8 @@ def action_send_and_print(self, force_synchronous=False, allow_fallback_pdf=Fals
             for move in self.move_ids:
                 if not move.peppol_move_state or move.peppol_move_state == 'ready':
                     move.peppol_move_state = 'to_send'
+        elif all([not self.checkbox_ubl_cii_xml, not self.checkbox_send_peppol, self.enable_peppol]):
+            self.move_ids.peppol_move_state = 'skipped'
```

Comment:
I think it's a very bad idea to write on moves here since you could have some raises after commit. Also, what is preventing you to select multiple moves with different peppol_move_state. You should update the state one by one when performing the peppol web service.
Why don't you set `to_send` in `_call_web_service_after_invoice_pdf_render` and `skipped` in `_hook_invoice_document_after_pdf_report_render`?

Reply:
I think `to_send` needs to remain here. Because it means "Queued" and it's supposed to show which ones are going to be sent asynchronously when sending multiple invoices at once. Skipped could be set in `_call_web_service_after_invoice_pdf_render` though?

Reply:
committed a counter suggestion :laughing: